### PR TITLE
[TASKMGR][TASKKILL][HIVESYS] Russian (ru-RU) translation update

### PR DESCRIPTION
**Pull request by Artem13327**
This message was written using Google Translate, sorry for any mistakes.
1. Translation of the word stub looked in Google Translate
2. The translation for "Error: Invalid syntax" was taken from /base/applications/cmdutils/tasklist/lang/ru-RU.rc
3. In one place, the translation option was chosen here https://chat.reactos.org/reactos/pl/t8oiwajkr7dymf4ck1qrra3y9w
4. **There is also one issue https://chat.reactos.org/reactos/pl/g7ij1rgq37re7nuohftry5b54h. Be sure to check if you accept this Pull Request**. Maybe I accidentally looked at something
5. I also ask that someone check this, otherwise I could be mistaken
6. I also want to thank GitHub for Сodespaces. This service has made my job a lot easier.
7. Made updates suggested in https://chat.reactos.org/reactos/pl/je86pmc5sfn69dw7ory4bcjmyy
8. Translation has become more like Windows XP (https://chat.reactos.org/reactos/pl/1bwg7zw4widomxj9gskbjoiksc and https://chat.reactos.org/reactos/pl/qu7yxfhjftn4tnfzre3cgpej9c)

### CODE REVIEW

**Comment by katahiromz** [comment]
> NO SQUASH MERGE

Ok.
■

**Comment by katahiromz** [comment]
Step by step.

**Comment by katahiromz** [comment]
Does this PR go?

**Comment by katahiromz** [comment]
You have to set your real name and public e-mail address to your commits and your account.

https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations

**Comment by katahiromz** [comment]
Cc @reactos/lang-russian

**Comment by katahiromz** [comment]
Ok, Russian guys will see. Thank you.

**Review by oleg-dubinskiy — APPROVED** [review]
Looks good to me. :slightly_smiling_face:

**Comment by oleg-dubinskiy** [comment]
But please, squash everything in a single commit. Then it will be easier to merge.

**Review by SigmaTel71 — APPROVED** [review]

File: base/applications/cmdutils/taskkill/lang/ru-RU.rc [hunk 1]
```diff
@@ -19,5 +19,5 @@ STRINGTABLE
     STRING_TERMINATE_FAILED, "Ошибка: Не удается завершить процесс ""%1"".\n"
     STRING_SELF_TERMINATION, "Ошибка: Процесс не может завершить сам себя.\n"
     STRING_PARAM_TOO_MUCH, "Error: Invalid syntax. '%1' option is not allowed more than '%2!d!' time(s).\n"
```
Comment (HBelusca) [inline base/applications/cmdutils/taskkill/lang/ru-RU.rc:21]:
```suggestion
    STRING_PARAM_TOO_MUCH, "Ошибка: Неверный синтаксис. Нельзя использовать опцию '%1' больше '%2!d!' раз(а).\n"
```

**Comment by HBelusca** [comment]
Manually squashed and committed since author's branch didn't allow in-place modification.

**Pull request closed without merging**